gradle/libs.versions.toml

```diff
@@ -41,6 +42,9 @@ slf4j = "2.0.0-alpha7"
 # Core EDC dependencies
 assertj = { module = "org.assertj:assertj-core", version.ref = "assertj" }
 awaitility = { module = "org.awaitility:awaitility", version.ref = "awaitility" }
+bouncyCastle-bcprov-jdk18on = { module = "org.bouncycastle:bcprov-jdk18on", version.ref = "bouncycastle-jdk18on" }
```

Comment:
AFAIK this will not work, because the "-" will act as a separator, thus generating a type-safe accessor named  `bouncyCastle.bcprov` and `bouncyCastle.bcpkix`, which would be  a leaf and a node at the same time, and that is not allowed.
In order to avoid conflicts, I would just remove the dash and capitalize the "J", which would result in `bouncyCastle-bcprovJdk18on` and `bouncyCastle-bcpkixJdk18on`